sme_finance_application_schema/lender_v3

```diff
@@ -0,0 +1,1665 @@
+{
```

Comment:
I thought lender_v3/4 are no longer to be in this repo?
Especially given that we've moved away from this structure internally